http/handlers.go

```diff
@@ -163,11 +163,11 @@ func ErrorHandler(logger log.Logger) func(context.Context, error, http.ResponseW
 		switch e := errors.Cause(err).(type) {
 		case security.ForbiddenError:
 			w.WriteHeader(http.StatusForbidden)
-			w.Write([]byte(errorhandler.GetEmitter() + "." + errorhandler.MsgErrOpNotPermitted))
+			_, _ = w.Write([]byte(errorhandler.GetEmitter() + "." + errorhandler.MsgErrOpNotPermitted))
 		case errorhandler.Error:
 			w.WriteHeader(e.Status)
 			// You should really take care of what you are sending here : e.Message should not leak any sensitive information
-			w.Write([]byte(e.Message))
+			_, _ = w.Write([]byte(e.Message))
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time